adminSDK/directory/index.js

```diff
@@ -18,19 +18,15 @@
 const fs = require('fs');
 const readline = require('readline');
 const {google} = require('googleapis');
-const GoogleAuth = require('google-auth-library');
 
-// If modifying these scopes, delete your previously saved credentials
-// at ~/.credentials/admin-directory_v1-nodejs-quickstart.json
+// If modifying these scopes, delete credentials.json.
```

Comment:
This should say to delete `token.json`, since that is where the token is stored. Looks like it's wrong in the other quickstarts as well. Please fix for the quickstarts in this PR at least.

Reply:
Oh right, will do